components/core/header/nav-bar/nav-bar-items.js

```diff
@@ -16,6 +16,10 @@ export default Object.freeze({
             i18nKey: 'codeOfConduct',
             value: '/about/code-of-conduct',
         },
+        {
```

Comment:
The link to this page is currently in the footer and doesn't need to be shown on the navbar. (see [figma](https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=4098%3A21164))